app/lib/frontend/templates.dart

```diff
@@ -191,6 +192,9 @@ class TemplateService {
         devDeps.isNotEmpty;
 
     final Map<String, dynamic> data = {
+      'package': package,
+      'show_outdated': analysis.analysisStatus == AnalysisStatus.outdated,
```

Comment:
Nit: Could also use `{{^show_content}}` in the template to avoid passing in two booleans for this state.

Reply:
Updated, although with the other flag (using `{{^show_outdated}}` in the template)